src/hotspot/cpu/x86/stubGenerator_x86_64.hpp

```diff
@@ -0,0 +1,450 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
As the content of this file has been moved from another, the original file's copyright years should be preserved. Thanks.

Reply:
Fixed.